### ISSUE TITLE
Update ruleset to segregate Security rules

### ISSUE THREAD

**Issue by ernilambar** [10up/plugin-check#643]
We should segregate `WordPress.Security.ValidatedSanitizedInput` and assign separate `type` and `severity` for each rules.

```
<rule ref="WordPress.Security.ValidatedSanitizedInput">
  <type>error</type>
  <severity>7</severity>
</rule>
```

**Comment by ernilambar** [comment]
`ValidatedSanitizedInput` has four sub rules:

* InputNotValidatedNotSanitized
* InputNotValidated
* InputNotSanitized
* MissingUnslash

It looks all WARNING to me. Looking for confirmation.

CC @frantorres

**Comment by davidperezgar** [comment]
I'd make `InputNotSanitized` for severity 7 and the others as 6. What do you think?

**Comment by ernilambar** [comment]
And what would be type for each item?

**Comment by frantorres** [comment]
> `ValidatedSanitizedInput` has four sub rules:
> 
> * InputNotValidatedNotSanitized
> * InputNotValidated
> * InputNotSanitized
> * MissingUnslash
> 
> It looks all WARNING to me. Looking for confirmation.

Looks like a Warning for me too as all of these can have situations in which false positives may arise.
